integration/standalone/webpack.config.js

```diff
@@ -50,7 +50,6 @@ module.exports = {
     new CircularDependencyPlugin({
       exclude: /(node_modules|examples)\/./,
       failOnError: false
-    }),
-    new webpack.WatchIgnorePlugin([/\.js$/, /\.d\.ts$/])
+    })
```

Comment:
Needed that we watch js and d.ts?

Reply:
No idea, somehow without this line the webpack will correctly refresh...
Maybe because we launch two watch services, one for the editor and one for the standalone. And the editor maybe recompiles the js files which need trigger the webpack or something like that?

Reply:
Ok makes sense!